lecture_2/homework/app/controllers/strongholds_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StrongholdsController < ApplicationController
+
+    def index
+        render json: Stronghold.all
+    end
+
+    def show 
+        render json: stronghold
+    end
+
+    def create
+        stronghold=stronghold.create(stronghold_params)
+    end
+
+
+
+    private 
+
+    def stronghold 
+        @stronghold ||= stronghold.find(params[:id])
```

Comment:
Tutaj podobnie. Pamiętaj, że `Stronghold` to jest model, a `stronghold` (pisany małą literą) to najprawdopodobniej będzie jakaś zmienna